src/utils/index.ts

```diff
@@ -165,4 +165,30 @@ export function loadAnalytics(factory: string): Analytics {
   }
 
   return analytics
+}
+
+export function loadAnalyticsHourData(factory: string, eventTimestamp: BigInt): AnalyticsHourData {
+  let timestamp = eventTimestamp.toI32()
+  let hourIndex = timestamp / 3600 // get unique hour within unix history
+  let hourStartUnix = hourIndex * 3600 // want the rounded effect
+  let analyticsHourID = factory
+    .concat('-')
+    .concat(hourIndex.toString())
```

Comment:
I understand the rounding. But `hourIndex` used for `id` is here just to have a smaller number in `id`? Because I don't see a reason to not use `hourStartUnix`  instead besides cosmetic reasons.